src/schema/v2/me/index.ts

```diff
@@ -226,13 +226,6 @@ const MeField: GraphQLFieldConfig<void, ResolverContext> = {
       "lotsByFollowedArtistsConnection",
       "identityVerification",
       "unreadNotificationsCount",
-      "receivePurchaseNotification",
-      "receiveOutbidNotification",
-      "receiveLotOpeningSoonNotification",
-      "receiveSaleOpeningClosingNotification",
-      "receiveNewWorksNotification",
-      "receiveNewSalesNotification",
-      "receivePromotionNotification",
```

Comment:
Oh man I totally misunderstood this code, I thought this was not required fields in metaphysics query. Good catch 🙇‍♂️ !